src/screens/ChangeWalletSheet.js

```diff
@@ -93,11 +94,11 @@ const ChangeWalletSheet = () => {
   }
 
   const onChangeAccount = useCallback(
-    async (wallet_id, address, fromDeletion = false) => {
```

Comment:
this was my bad. I think we can add enable this eslint if you want: https://eslint.org/docs/rules/camelcase